tests/pass-rate.sh

```diff
@@ -22,8 +27,8 @@ successes=$(cat status.out | grep -c success)
 let total=$successes+$failures
 passrate=$(bc -l <<< "scale=2;100 * ($successes / $total)")
```

Comment:
will this now have a chance to fail with `Divide by zero` error at the beginning of the sprint when there are no `$total` builds?

Reply:
Good catch. I'll make sure we handle that case!